tests/filterNew.test.js

```diff
@@ -0,0 +1,61 @@
+import {subDays, subMonths, subWeeks} from "date-fns";
+import {filterNew} from "../src/lib";
+
+describe('filterNew()', () => {
+
+  const date1 = subMonths(new Date(), 3).toISOString()
+  const date2 = subMonths(new Date(), 1).toISOString()
+  const date3 = subWeeks(new Date(), 2).toISOString()
+  const date4 = subDays(new Date(), 2).toISOString()
+  const date5 = subDays(new Date(), 1).toISOString()
+  const todayDate = new Date()
```

Comment:
You might want to define todayDate once and then reuse it – just to make sure you don't have a mismatch (say if you run it at midnight and the date actually changes during the execution)
```suggestion
  const todayDate = new Date()
  const date1 = subMonths(todayDate, 3).toISOString()
  const date2 = subMonths(todayDate, 1).toISOString()
  const date3 = subWeeks(todayDate, 2).toISOString()
  const date4 = subDays(todayDate, 2).toISOString()
  const date5 = subDays(todayDate, 1).toISOString()

```

Reply:
It might also be good to have more descriptive names for the dates – things like `threeMonthsAgo`, `oneMonthAgo`, `twoDaysAgo`, `oneWeekAgo` etc., so the test cases are easier to comprehend without referring to the definitions.